.github/workflows/chart.yaml

```diff
@@ -6,11 +6,39 @@ on:
       - 'main'
     paths:
       - 'charts/**'
+  pull_request:
+    branches:
+      - 'main'
+    paths:
+      - 'charts/**'
 
 jobs:
+  helm-verify:
+    runs-on: ubuntu-latest
+    if: github.event_name != 'push'
```

Comment:
I think this should only run on `pull_request` as you're explicitly referencing that in the `old_version` step to get the head sha